api/v1/db/oracledb/game-dao.js

```diff
@@ -0,0 +1,63 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+const { serializeGames } = require('../../serializers/games-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+/**
+ * @summary Merge the raw response from database. Extract card information from every row,
+ * and put them into a field called 'tableCard' in the individual merged game object, while other
+ * properties in the object remained in the first layer.
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of games
+ */
+const mergeRawGames = (rawGames) => {
+  const groupedRawGames = _.groupBy(rawGames, 'GAME_ID');
+  const mergedRawGames = _.map(groupedRawGames, (gameMetaDataArray) => {
+    gameMetaDataArray[0].tableCards = gameMetaDataArray[0].CARD_NUMBER === null ? []
+      : _.map(gameMetaDataArray, data => ({
+        cardNumber: data.CARD_NUMBER,
+        cardSuit: data.SUIT,
+      }));
+    return gameMetaDataArray[0];
+  });
+  return mergedRawGames;
+};
+
+/**
+ * @summary Return a list of games
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of games
+ */
```

Comment:
Could you add `@param` to this function and `mergeRawGames`? I think in VS code, you can add some starter documentation by placing your cursor on the line above the function definition and typing `/**[enter]` which can be helpful